packages/core/fs/src/index.js

```diff
@@ -66,11 +66,12 @@ export const ncp: (
 export function writeFileStream(
   filePath: FilePath,
   stream: Readable
-): Promise<void> {
+): Promise<number> {
```

Comment:
does this really need to return a number? since we have a reference to the stream in the `PackagerRunner` (or any other case where we'd call this), we could just read this property directly after this resolves.

This is fine too, though maybe we could return something more meaningful in the future.

Reply:
> since we have a reference to the stream in the PackagerRunner (or any other case where we'd call this), we could just read this property directly after this resolves.

No, `bytesWritten` exists only on the FS.WriteStream, not on the "normal" Writeable stream. And `fs.createWriteStream(filePath))` is called in this function

Reply:
ah yeah. Sounds good.